libraries/README.md

```diff
@@ -0,0 +1,8 @@
+# RightScale Policy Templates Libraries and Examples
+
+This folder contains a set of libraries and examples that can be used to aid in the building of policy templates.  All contributions are shared under the MIT license.
+
```

Comment:
I suggest adding some instructions.  It should indicate that they must copy the examples into their policies and adjust to meet their needs.  
They may assume that can do some CAT imports or something.